test/Microsoft.AspNet.JsonPatch.Test/NestedObjectTests.cs

```diff
@@ -281,7 +281,10 @@ public void AddToListInvalidPositionTooLarge()
             patchDoc.Add<int>(o => o.SimpleDTO.IntegerList, 4, 4);
 
             // Act & Assert
-            Assert.Throws<JsonPatchException<SimpleDTOWithNestedDTO>>(() => { patchDoc.ApplyTo(doc); });
+            var exception = Assert.Throws<JsonPatchException<SimpleDTOWithNestedDTO>>(() => { patchDoc.ApplyTo(doc); });
+            Assert.Equal("For Operation 'add', array property at location path: '/simpledto/integerlist/4' index is " +
+                "larger than array size.", exception.Message);
```

Comment:
let's try to simplify this message and boil it down to something more readable.

Suggestion:
```
"For Operation 'add' on array property '/simpledto/integerlist/4', the index is larger than the array size."
```